server/util.go

```diff
@@ -173,6 +173,10 @@ func CreateSDKLogger(ctx *Context, out io.Writer) (log.Logger, error) {
 	if ctx.Viper.GetString(flags.FlagLogFormat) == flags.OutputFormatJSON {
 		opts = append(opts, log.OutputJSONOption())
 	}
+	opts = append(opts,
+		log.ColorOption(ctx.Viper.GetBool(flags.FlagLogColored)),
+		// CometBFT flag (cmtcli.TraceFlag) for trace logging.
```

Comment:
I still find the current wording too unclear as you may think it is for log level trace.
While it is for logging stack traces in errors.